neuromation/api/quota.py

```diff
@@ -42,17 +41,23 @@ def __init__(self, core: _Core, config: Config) -> None:
             return _quota_info_from_api(res)
 
 
-def _quota_info_from_api(payload: Dict[str, Any]) -> _QuotaInfo:
-    jobs = payload["jobs"]
-    spent_gpu = float(int(jobs["total_gpu_run_time_minutes"]) * 60)
-    spent_cpu = float(int(jobs["total_non_gpu_run_time_minutes"]) * 60)
-    quota = payload["quota"]
-    limit_gpu = float(quota.get("total_gpu_run_time_minutes", "inf")) * 60
-    limit_cpu = float(quota.get("total_non_gpu_run_time_minutes", "inf")) * 60
-    return _QuotaInfo(
-        name=payload["name"],
-        gpu_time_spent=spent_gpu,
-        gpu_time_limit=limit_gpu,
-        cpu_time_spent=spent_cpu,
-        cpu_time_limit=limit_cpu,
-    )
+def _quota_info_from_api(payload: Dict[str, Any]) -> Dict[str, _QuotaInfo]:
+    clusters = payload["clusters"]
+    ret: Dict[str, _QuotaInfo] = {}
+    for cluster_payload in clusters:
+        cluster_name = cluster_payload["name"]
+        jobs_payload = cluster_payload["jobs"]
+        quota_payload = cluster_payload.get("quota", {})
```

Comment:
looks like `quota` is always present in the response: https://github.com/neuromation/platform-api/blob/115c3f07e02356aec14fb473f71b799095f934d7/platform_api/handlers/stats_handler.py#L96

Reply:
but might be `{}` an empty dict, yep.

Reply:
There was test for missed quota before. Then I add this case. Can I safetly remove or better to leave for future?

Reply:
we can leave as is. up to you.